src/App.jsx

```diff
@@ -14,33 +16,92 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const SORT_FIELD_ALPHABET = 'string';
+const SORT_FIELD_LENGTH = 'length';
```

Comment:
Since `string` does not reflect the true nature of alphabetical sorting. Better to use `alphabet` for clarity.

```suggestion
const SORT_FIELD_ALPHABET = 'alphabet';
const SORT_FIELD_LENGTH = 'length';
```